
display_names:
  - 'ff'
  - 'mf'
  - 'rf'
  - 'lf'
  - 'th'
sensor_touch_names:
  - 'FF_Touch'
  - 'MF_Touch'
  - 'RF_Touch'
  - 'LF_Touch'
  - 'TH_Touch'
sensor_temp_names:
  - 'FF_Temp'
  - 'MF_Temp'
  - 'RF_Temp'
  - 'LF_Temp'
  - 'TH_Temp'
